### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by KingSaim0n** [Snownee/Kiwi#14]
* Forge Version: 31.2.5
 * Mod Version: 2.7.1

Paste your crash report to somewhere like https://paste.ubuntu.com/ , instead of directly here.
https://paste.ubuntu.com/p/NvKdzc53yZ/
![Untitled](https://user-images.githubusercontent.com/65935837/82858971-f92a1300-9f47-11ea-8def-bff18dd6a381.png)

**Comment by Lucasd10** [comment]
I am having that same issue I hope they fix it but for now, I changed to forge version 31.2.0 the recommended one and it works and I don't get that error.

**Comment by KingSaim0n** [comment]
Glad to see you here. I might just do what you just said. Thank you very much

**Comment by Lucasd10** [comment]
yup no problem I was searching for the same error and I saw your error and I signed up for github because I knew I had to respond to see if there was a fix.
